scripts/Makefile.build

```diff
@@ -307,25 +307,25 @@ $(obj)/%.lst: $(src)/%.c FORCE
 # ---------------------------------------------------------------------------
 
 quiet_cmd_cargo = CARGO $(quiet_modtag) $@
-      cmd_cargo = export RUST_BINDINGS_FILE=$(shell readlink -f $(srctree)/include/generated/rust_bindings.rs) && cd $(src) && $(CARGO) build $(cargo_flags)
+      cmd_cargo = export RUST_BINDINGS_FILE=$(shell readlink -f include/generated/rust_bindings.rs) && $(CARGO) build --manifest-path $(srctree)/$(src)/Cargo.toml --out-dir $(src) $(cargo_flags)
 
-$(srctree)/include/generated/rust_bindings.rs: FORCE
+include/generated/rust_bindings.rs: FORCE
```

Comment:
I am not really sure when `$(srctree)` is really needed, but did something break with it?

Reply:
Yes, if O is build, srctree is .., so it breaks

Reply:
Ah, it is relative, I see!